src/EnergyPlus/FluidProperties.cc

```diff
@@ -4944,42 +2667,42 @@ namespace FluidProperties {
         static constexpr std::string_view RoutineName("InterpDefValuesForGlycolConc: ");
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int HiIndex;       // index on the high side of the concentration
+        size_t HiIndex;    // index on the high side of the concentration
         Real64 InterpFrac; // intermediate value for interpolations
-        int LoopC;         // loop counter for concentration
-        int LoopT;         // loop counter for temperature
+        size_t LoopC;      // loop counter for concentration
+        size_t LoopT;      // loop counter for temperature
 
         // First, find where the actual concentration falls between the concentration data.
         // Then, interpolate if necessary.
-        if (Concentration < RawConcData(1)) { // Concentration too low
+        if (Concentration < RawConcData[0]) { // Concentration too low
             ShowWarningError(state,
                              format("{}Glycol concentration out of range for data (too low), concentration = {:.3R}", RoutineName, Concentration));
             ShowContinueError(state, "Check your data or the definition of your glycols in the GlycolConcentrations input");
             ShowContinueError(state, "Property data set to data for lowest concentration entered");
-            InterpData = RawPropData(_, 1);
-        } else if (Concentration > RawConcData(NumOfConcs)) { // Concentration too high
+            InterpData = RawPropData[0][0];
+        } else if (Concentration > RawConcData[NumOfConcs - 1]) { // Concentration too high
             ShowWarningError(state,
                              format("{}Glycol concentration out of range for data (too high), concentration = {:.3R}", RoutineName, Concentration));
             ShowContinueError(state, "Check your data or the definition of your glycols in the GlycolConcentrations input");
             ShowContinueError(state, "Property data set to data for highest concentration entered");
-            InterpData = RawPropData(_, NumOfConcs);
-        } else {                  // Concentration somewhere between lowest and highest point--interpolate
-            HiIndex = NumOfConcs; // Default to highest concentration
-            for (LoopC = 2; LoopC <= NumOfConcs - 1; ++LoopC) {
-                if (Concentration <= RawConcData(LoopC)) {
+            InterpData = RawPropData[NumOfConcs - 1][0];
```

Comment:
To access data from `std::array<std::array>`, we use `arrayName[columnIndex][rowIndex]`. For Objexx `Array2S`, this is `arrayName(rowIndex,columnIndex)`.